server/etc/pulp/server.conf

```diff
@@ -101,6 +101,9 @@
 # ssl_ca_certificate: full path to the CA certificate used to sign the Pulp
 #     server's SSL certificate; consumers will use this to verify the
 #     Pulp server's SSL certificate during the SSL handshake
+#     Deprecated! - Please note that this setting will be removed in the next
+#     release and will be replaced by a setting in the consumer.conf file
```

Comment:
Might be nice to specify the next X release since this would not be removed in a hypothetical 2.7.